src/components/SearchResults/SearchResults.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { describe, it, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "./SearchResults.jsx";
+import FakeBookings from "../../data/fakeBookings.json";
+
+describe("SearchResults Component", () => {
+  it("renders a table element", () => {
+    render(<SearchResults bookings={FakeBookings} />);
+    const searchResultsElement = screen.getByTestId("search-results-component");
+    expect(searchResultsElement).toBeInTheDocument();
+  });
+
+  it("renders booking components for each booking", () => {
+    render(<SearchResults bookings={FakeBookings} />);
+    const bookingComponents = screen.getAllByTestId("booking-component");
+    expect(bookingComponents.length).toBe(FakeBookings.length);
+  });
+});
```

Comment:
It depends on the test. In this scenario, for example, the test checks if the number of bookingComponents rendered is the same as the number of bookings in the FakeBookings array. It means that the page is rendering all the bookings correctly.
Of course, when we implement the search feature, this test might be changed to verify if the rendered amount matches the filtered array (after search) rather than the original array with all bookings.

Reply:
ohhh okay, this makes sense now. Thank you for your explanation.